openshift_automation_drafts.txt

```diff
@@ -0,0 +1,28 @@
+ ./hack/update-crds.sh --> generateCRD
```

Comment:
Is this documentation?  Please rename to a markdown file.

Reply:
I removed also this... it was only a draft for csv generation , and image bundle creation on openshift